client/verta/verta/operations/monitoring/summaries.py

```diff
@@ -261,6 +287,28 @@ def create(self, name, data_type_cls, monitored_entity):
         proto = self._conn.must_proto_response(response, SummaryProto)
         return Summary(self._conn, self._conf, proto)
 
+    def get_or_create(self, name, data_type_cls, monitored_entity):
+        assert issubclass(data_type_cls, data_types._VertaDataType)
```

Comment:
Raising an explicit `TypeError` may be preferrable (particularly since an `AssertionError` doesn't include a plain-English error message)